### ISSUE TITLE
Update to Angular Material 10

### ISSUE THREAD

**Issue by FloBam** [kuhnroyal/mat-datetimepicker#108]
Hi, [Angular 10 including Material 10](https://blog.angular.io/version-10-of-angular-now-available-78960babd41) has been released for a few days.

when running `ng update @angular/material` you will get the following:

```
Package "@mat-datetimepicker/core" has an incompatible peer dependency to "@angular/material" (requires "~9.2.1", would install "10.0.1").
Package "@mat-datetimepicker/core" has an incompatible peer dependency to "@angular/cdk" (requires "~9.2.1", would install "10.0.1").
Package "@mat-datetimepicker/moment" has an incompatible peer dependency to "@angular/material-moment-adapter" (requires "~9.2.1", would install "10.0.1").
```

Would be great if there was a release supporting the new material version :slightly_smiling_face:

**Comment by sergidt** [comment]
Totally agree. We cannot update our projects to angular 10 due we are using @mat-datetimepicker/core and requires angular 9

**Comment by jjtortosa** [comment]
You can use --force:  ng update @angular/material --force
It worked for me

**Comment by sergidt** [comment]
Thank you @jjtortosa! i'll check it!

**Comment by SamuelMarks** [comment]
Just opened a PR for this #114

**Comment by bbortt** [comment]
it's released in `5.0.0` and a minor `README.md` bugfix in `5.0.1`. thanks a lot @SamuelMarks.
